exporter/datadogexporter/config/config.go

```diff
@@ -205,13 +205,22 @@ func (t *TagsConfig) GetHostTags() []string {
 	return tags
 }
 
+// LimitedTLSClientSetting is a subset of TLSClientSetting, see LimitedHTTPClientSettings for more details
+type LimitedTLSClientSetting struct {
+	// InsecureSkipVerify controls whether a client verifies the server's
+	// certificate chain and host name.
+	InsecureSkipVerify bool `mapstructure:"insecure_skip_verify"`
+}
```

Comment:
The config struct is considered part of the public API and therefore we won't be able to make breaking changes to it in the future. To future-proof, we should put this in a `LimitedHTTPClientSettings` like

```go
type LimitedHTTPClientSettings struct {
	TLSSetting LimitedTLSClientSettings `mapstructure:"tls,omitempty"`
}
```

and then embed it on the `Config` struct:

```go
LimitedHTTPClientSettings `mapstructure:",squash"`
```

This results in the same YAML configuration but we future-proof the Go API too.